pontoon/base/models.py

```diff
@@ -1031,6 +1043,23 @@ def get_details(parts):
 
 
 class ProjectQuerySet(models.QuerySet):
+    def visible_for(self, user):
+        """
+        The visiblity of projects is determined by the role of the user:
+        * Administrators can access all public and private projects
+        * Other users can see only public projects
+        """
+        if user.is_superuser:
+            return self
+
+        return self.public()
+
+    def public(self):
```

Comment:
We don't seem to need this method. I'll leave it to you to decide if we should still keep it.